contracts/base/DAIPool.sol

```diff
@@ -0,0 +1,139 @@
+/*
+    Copyright 2020 Fabrx Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+pragma solidity 0.5.17;
+
+// Libraries
+import "../util/AddressLib.sol";
+
+// Commons
+import "../util/ZeroCollateralCommon.sol";
+
+// Interfaces
+import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import "../interfaces/DAIPoolInterface.sol";
+import "../interfaces/LenderInfoInterface.sol";
+import "../interfaces/LoansInterface.sol";
+import "../interfaces/ZDAIInterface.sol";
+
+// Contracts
+import "./Initializable.sol";
+
+
+contract DAIPool is DAIPoolInterface, Initializable {
+    using AddressLib for address;
+
+    /* State Variables */
+
+    IERC20 public dai;
+
+    ZDAIInterface public zdai;
+
+    LenderInfoInterface public lenderInfo;
+
+    LoansInterface public loanInfo;
+
+    /** Modifiers */
+
+    /* Constructor */
+
+    /** External Functions */
+
+    function initialize(
+        address zdaiAddress,
+        address daiAddress,
+        address lenderInfoAddress,
+        address loanInfoAddress
+    ) external isNotInitialized() {
```

Comment:
The code is inconsistent whether it uses parentheses on modifiers. e.g. line 59 uses `isNotInitialized()` and 78 uses `isInitialized`

Reply:
You're right, I have added the parentheses on them.